src/nrnpython/nrnpy_nrn.h

```diff
@@ -115,6 +115,24 @@ static PyType_Spec nrnpy_MechanismType_spec = {
     nrnpy_MechanismType_slots,
 };
 
+static PyType_Slot nrnpy_MechFuncType_slots[] = {
+    {Py_tp_dealloc, (void*) NPyMechFunc_dealloc},
+    {Py_tp_repr, (void*) pymechfunc_repr},
+    {Py_tp_methods, (void*) NPyMechFunc_methods},
+    {Py_tp_call, (void*) NPyMechFunc_call},
+    //    {Py_tp_init, (void*) NPyMechFunc_init},
+    //    {Py_tp_new, (void*) NPyMechFunc_new},
```

Comment:
these are possible placeholders and hence kept here?

Reply:
Appears to be an oversight. This started existence as a copy of the ```nrnpy_MechType_slots``` fragment and then every occurrence of MechType was changed to MechFuncType.  At the time I did not know if I would need the init and new methods so commented them out. Apparently they are not needed so I'll remove them.